cmd/fetch.go

```diff
@@ -124,14 +130,15 @@ Pass Zookeeper nodes separated by a space as an argument to this command.`,
 }
 
 var mesosCmd = &cobra.Command{
-	Use:               "mesos [zkNode0, zkNode1, ...zkNodeN]",
+	Use:               "mesos leader [zkNode0, zkNode1, ...zkNodeN]",
```

Comment:
Will this work how we expect it to? I remember I had to make a command (mesos in this case) which did nothing and attached the subcommand (leader) to it.

This may work though not sure.

Reply:
Good point, will test after making initial changes.